### DISCUSSION TITLE
Welcome to listen-my-order Discussions!

### DISCUSSION THREAD

**Discussion by ycs1m1yk** [euphony-io/listen-my-order#45] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ycs1m1yk** [comment]
discussion 페이지 열었습니다!! 🎉 🎉
@HACore @hoho00 @YoungSeokHong

**Reply by HACore** [reply]
바로 열어주셨네요!! 감사합니다~😄
